doc/bt/BC_NORMAL.bt

```diff
@@ -81,14 +81,33 @@ switch(type)
     case 0x0C:
         byte bin1[size-12]; // zipped zone information
         break;
+
+    // Enable packets from client to barrack.
+    // (CB packets are disabled normally after connecting to the zone server.)
```

Comment:
This might be necessary when visiting barracks of other players, but I'm not certain.

Reply:
That comment should've made it into the template^^